src/rpdk/core/contract/suite/handler_create.py

```diff
@@ -43,6 +44,12 @@ def contract_create_delete(resource_client):
         test_delete_success(resource_client, delete_model)
 
 
+@pytest.mark.create
+def contract_invalid_create(resource_client):
+    requested_model = resource_client.generate_invalid_create_example()
+    test_create_fail_if_read_only_property_in_input(resource_client, requested_model)
```

Comment:
i know we have this "handler commons" module but honestly if this test is only used once might as well inline it here for clarity. no reason to have someone jump to another file to get the gist